docs/usage/node.md

```diff
@@ -33,3 +33,7 @@ For example, if you want to use at least npm `8.1.0` and also allow newer versio
   }
 }
 ```
+
+## LTS codenames
```

Comment:
How about moving the new section up a bit? Like this:

```markdown
# Node.js Versions
Renovate can upgrade the [Node.js](https://nodejs.org/en/) runtime used by your project.
This way you're using the latest bug fixes, performance improvements, security mitigations, etc.

## LTS codenames

Renovate understands [codenames for Node.js LTS releases](https://github.com/nodejs/Release/blob/main/CODENAMES.md) and will offer upgrades for them (e.g. from `fermium` to `gallium`).
```

This way you get a general introduction first and right after that you'll learn about the Node.js codename support.

Reply:
Yes, that reads a bit better! Done with c38a61933